wagtail/tests/testapp/models.py

```diff
@@ -1176,7 +1219,8 @@ class StreamPage(Page):
                     ]
                 ),
             ),
-        ]
+        ],
+        use_json_field=False,
```

Comment:
Should we also duplicate the tests for pages that use `StreamField`? That would be a lot of grounds to cover and I'm not sure if it's worth doing.

For the record, I tried flipping `use_json_field` to `True` for all the `Page` models here and all the tests passed.